about.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About_Africa's talking</title>
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+<body>
+    <nav class="navbar">
+        <div class="top-nav">
+            <ul class="top-nav-list">
+                <li><a href="#" class="fa fa-facebook"></a></li>
+                <li><a href="#" class="fa fab fa-twitter"></a></li>
+                <li><a href="#" class="list">Logout</a></li>
+                <li><a href="#" class="list">Mypage</a></li>
+                <li><a href="#" class="list">English</a></li>    
+            </ul>
+        </div>
+        <div class="navbar-mobile">
+            <div class="logo">
+                <a href="index.html"><img class="logo-image" src="images/logo.svg"></a>
+            </div>
+            <div class="menu-icon">
+                <a id="show-menu" class="fa fa-bars" href="#"></a>
+            </div>
+            <div class="navbar-desktop invisible">
+                <ul class="desktop-menu-list">
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="#">Events</a></li>
+                    <li><a href="#">Documentation</a></li>
+                    <li><a href="about.html">About</a></li>    
+                </ul>
+            </div>
+        </div>
```

Comment:
- [ ] I am so proud that you've used the previous reviewer's advice of starting from the ground and reconstructing your project in an improved version, but again you've missed small things, especially your font styling. Most of your texts have a very huge different size differ from the provided design that you should refer to. 
Pass through your webpage and compare each part of your project to the one from behance design. 
![image](https://user-images.githubusercontent.com/43877689/175770904-5e34d04f-6a5e-4621-bc45-d070a5a4eddb.png)
============== Behance design ==========================
![image](https://user-images.githubusercontent.com/43877689/175770945-d81311a0-89f3-4b25-89d3-e6840474905e.png)

- [ ]  Its seemed that from your design navigation, you didn't add **CC Compain** part, and you should have that element in you navigations as well, so that you design can look closer to the one provided from Behance design.